StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -526,30 +526,51 @@ def copy_icon(md5, icon_path=''):
 
 def get_app_name(app_path, app_dir, tools_dir, is_apk):
     """Get app name."""
-    data = ''
     if is_apk:
         a = apk.APK(app_path)
         real_name = a.get_app_name()
         return real_name
     else:
         strings_path = os.path.join(app_dir,
-                                    'app/src/main/res/values/strings.xml')
+                                    'app/src/main/res/values/')
         eclipse_path = os.path.join(app_dir,
-                                    'res/values/strings.xml')
+                                    'res/values/')
         if os.path.exists(strings_path):
-            strings_file = strings_path
+            strings_dir = strings_path
         elif os.path.exists(eclipse_path):
-            strings_file = eclipse_path
-    if not os.path.exists(strings_file):
-        logger.warning('Cannot find app name')
+            strings_dir = eclipse_path
+    if not os.path.exists(strings_dir):
+        logger.warning('Cannot find values folder.')
         return ''
 
-    with open(strings_file, 'r', encoding='utf-8') as f:
+    # Return found app_name or 'None' if it didn't find it.
+    return get_app_name_from_values_folder(strings_dir) or 'None'
```

Comment:
This is not consistent with other `returns` consider changing this to `return get_app_name_from_values_folder(strings_dir)`